app/models/conversation.rb

```diff
@@ -0,0 +1,16 @@
+class Conversation < ActiveRecord::Base
+  belongs_to :sender, :foreign_key => :sender_id, class_name: 'User'
+  belongs_to :recipient, :foreign_key => :recipient_id, class_name: 'User'
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.